index.html

```diff
@@ -86,201 +86,247 @@ <h1>
+                        </ul>
+                        <a href="#" class="check-button">Get my resume</a>
+                    </div>
+                    <div class="skill-list">
+                        <ul>
+                            <li>
+                                <span class="skills">
+                                    Languages <span><img src="./images/Disabled.png" alt="languages"></span>
                                 </span>
+                                <ul class="skills-languages">
+                                    <li class="add-skills">
+                                        <a href="#"><img src="./images/Ellipse 1.png" alt="JS"></a>
+                                        <span class="text-skills">
+                                            JavaScript
+                                        </span>
+                                    </li>
+                                    <li class="add-skills">
+                                        <a href="#"><img src="./images//Ellipse 1 (1).png" alt="HTML"></a>
+                                        <span class="text-skills">
+                                            HTML
+                                        </span>
+                                    </li>
+                                    <li class="add-skills">
+                                        <a href="#"><img src="./images//Ellipse 1 (2).png" alt="CSS"></a>
+                                        <span class="text-skills">
+                                            CSS
+                                        </span>
+                                    </li>
+                                </ul>
+                            </li>
+                            <li class="skills">Frameworks <span><img src="./images/Enabled.png" alt="frameworks"></span>
+                            </li>
+                            <li>
+                                <hr class="hr-line">
+                            </li>
+                            <li class="skills">Skills <span><img src="./images/Enabled.png" alt="Skills"></span></li>
+                            <li>
+                                <hr class="hr-line">
                             </li>
                         </ul>
-                    </li>
-                    <li class="skills">Frameworks <span><img src="./images/Enabled.png" alt="frameworks"></span></li>
-                     <li>
-                        <hr class="hr-line">
-                     </li>
-                    <li class="skills">Skills <span><img src="./images/Enabled.png" alt="Skills"></span></li>
-                    <li>
-                        <hr class="hr-line">
-                    </li>
-                </ul>
+                    </div>
+
+
+
+                </section>
+
+                <!-- 
+                contact form -->
+
+                <section class="contact-me">
+                    <img class="contact-bg-image" alt="Image missing">
```

Comment:
- [ ] This section's background image should be placed using CSS and not HTML element as you have done here. If an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.